build.gradle.kts

```diff
@@ -524,7 +539,7 @@ afterEvaluate {
 			doLast {
 				val http = HttpUtils()
 
-				val currentTag = releaseTags.getOrNull(0)
+				val currentTag: Tag? = releaseTags.getOrNull(0)
```

Comment:
This is why inferred typing is not an excuse for turing-style typing specification.

Reply:
the linter broke on the `!= null` check 2 lines before this and adding this fixed it
my kotlin may just be broken though

Reply:
My point was turing-style typing likes to use the excuse of "you usually don't need to specify types anyway". Anyway not your doing it's Kotlin's problem.